pkg/state/state.go

```diff
@@ -221,6 +223,7 @@ type RepositorySpec struct {
 	Keyring         string `yaml:"keyring,omitempty"`
 	PassCredentials bool   `yaml:"passCredentials,omitempty"`
 	SkipTLSVerify   bool   `yaml:"skipTLSVerify,omitempty"`
+	PlainHttp       bool   `yaml:"plainHttp,omitempty"`
```

Comment:
where this field be used?

Reply:
It is used here: https://github.com/helmfile/helmfile/pull/1672/files/bab7cd3092d9d25cb2a4fb476a81d9a9737ae3fd#diff-7e4e87a1e777423a3d9a915e00fb78a9c4fc2f4c476863f60b67e89b9c635697R2864-R2866

Which is used to pull charts from OCI repositories in `getOCIChart` in `PrepareCharts`.